playground/mocks/config/responseConfig.js

```diff
@@ -139,6 +139,9 @@ const idx = {
   '/idp/idx/activate': [
     'identify-with-password',
     // 'error-invalid-device-code',
+  ],
+  '/idp/idx/authenticators/sso_extension/transactions/:transactionId/verify/cancel': [
```

Comment:
How does the test get fixed by adding it to this file ?  From what I remember you will need to make sure the mocks response is set for this call in the EnglishLeaks_spec https://github.com/okta/okta-signin-widget/blob/97ace00b7dbece58ff7b9e4f91128af3fb218f06/test/testcafe/spec-en-leaks/EnglishLeaks_spec.js#L53